pkg/kubelet/cm/topologymanager/topology_manager.go

```diff
@@ -134,6 +134,10 @@ func NewManager(numaNodeInfo cputopology.NUMANodeInfo, topologyPolicyName string
 		return nil, fmt.Errorf("unsupported on machines with more than %v NUMA Nodes", maxAllowableNUMANodes)
 	}
 
+	if topologyPolicyName != PolicyNone && len(numaNodes) == 1 {
+		return nil, fmt.Errorf("%s is unsupported on the machine has only single NUMA node", topologyPolicyName)
```

Comment:
This makes sense to me, especially if/when we move from `none` to `best-effort` as the default policy.

The log message is a bit confusing though. For instance, under the `single-numa-node` policy, this would read as: `single-numa-node is unsupported on the machine has only single NUMA node`

Perhaps something more straightforward like:
`fmt.Errorf("unsupported on machines with only one NUMA Node")`